src/main/java/life/qbic/io/commandline/PostmanCommandLineOptions.java

```diff
@@ -147,7 +147,7 @@ void listDatasets(
 
   @Option(names = {"-s", "--suffix"},
       split = ",",
-      description= "only include files ending with one of these suffixes",
+      description= "only include files ending with one of these (case-insensitive) suffixes",
```

Comment:
Major NIT: I don't think you need to specify that they can be case insensitive here